### DISCUSSION TITLE
Could not load model after best_model and last_model have been saved as .keras

### DISCUSSION THREAD

**Discussion by ocinei** [aeon-toolkit/aeon#1390] [Q&A]
Hello. After the latest update in 0.8.0 that saves best_model and last_model in ".keras" format, 
`tf.keras.models.load_model("path_to_keras_file", compile=False)` will raise an error. 

Some of the errors (for individual inception time model):
`Layer 'conv1d_5' expected 1 variables, but received 0 variables during loading. Expected: ['conv1d_5/kernel:0']
`

**Comment by MatthewMiddlehurst** [comment]
Thanks for the post. @hadifawaz1999 any idea?

**Comment by hadifawaz1999** [comment]
which tensorflow version you have on your system ? can you provide the full code you're using please ?
